p2-repository/pom.xml

```diff
@@ -0,0 +1,359 @@
+
+    <!-- for Orbit, by default we'll also remove the Require-Bundle header; recipe poms may override if they need this header -->
+    <recipe.removeadditionalheaders>Require-Bundle</recipe.removeadditionalheaders>
+
+    <!-- read BND settings from 'osgi.bnd' located next to pom.xml -->
+    <recipe.bndfile>osgi.bnd</recipe.bndfile>
+
+    <!-- pattern/format used for the qualifier -->
+    <recipe.qualifier.format>'v'yyyyMMdd-HHmm</recipe.qualifier.format>
+
+    <!-- project defaults -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- versions used for plug-ins below -->
+    <tycho-version>1.0.0</tycho-version>
+    <tycho-extras-version>1.0.0</tycho-extras-version>
+    <maven-resources-version>2.7</maven-resources-version>
+    <maven-dependency-version>2.10</maven-dependency-version>
+
+    <!-- versions of the EBR plug-in to use -->
+    <ebr-version>1.0.0-SNAPSHOT</ebr-version>
+
+    <!-- property for generating Eclipse source reference bundle headers -->
+    <tycho.scmUrl>scm:git:https://github.com/eclipse/eclipse-collections.git</tycho.scmUrl>
+
+    <!-- by default complain if working tree is dirty (error|warning|ignore) -->
+    <dirtyWorkingTree>warning</dirtyWorkingTree>
+
+    <!-- set default vendor -->
+    <bundleVendor>Eclipse Collections</bundleVendor>
+
+    <!-- values needed for generating repositories -->
+    <maven.build.timestamp.format>yyyyMMddHHmmss</maven.build.timestamp.format>
+    <buildTimestamp>${maven.build.timestamp}</buildTimestamp>
+    <buildType>I</buildType>
+    <buildId>${buildType}${buildTimestamp}</buildId>
+    <buildLabel>${buildType}-${buildId}-${buildTimestamp}</buildLabel>
+
+  </properties>
+
+  <pluginRepositories>
+    <!-- configure the EBR plug-in repo -->
+    <pluginRepository>
+        <id>ebr-releases</id>
+        <url>https://repo.eclipse.org/content/repositories/ebr-releases/</url>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+    </pluginRepository>
+    <pluginRepository>
+        <id>ebr-snapshots</id>
+        <url>https://repo.eclipse.org/content/repositories/ebr-snapshots/</url>
+        <releases>
+            <enabled>false</enabled>
+        </releases>
+    </pluginRepository>
+    <pluginRepository>
+        <id>cbi-releases</id>
+        <url>https://repo.eclipse.org/content/repositories/cbi-releases/</url>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+    </pluginRepository>
+    <pluginRepository>
+        <id>cbi-snapshots</id>
+        <url>https://repo.eclipse.org/content/repositories/cbi-snapshots/</url>
+        <snapshots>
+            <enabled>true</enabled>
+        </snapshots>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <repositories>
+  </repositories>
+
+  <profiles>
+    <profile>
+      <id>eclipse-sign</id>
```

Comment:
Our release-artifact profile already consists the jar sign step, can we merge both of them?
https://github.com/eclipse/eclipse-collections/blob/master/pom.xml#L656